src/onINP.ts

```diff
@@ -84,28 +158,37 @@ const processEntry = (entry: PerformanceEventTiming) => {
     longestInteractionList.length < MAX_INTERACTIONS_TO_CONSIDER ||
     entry.duration > minLongestInteraction.latency
   ) {
+    const entriesGroup = pendingEntriesGroupMap[renderTime];
+
     // If the interaction already exists, update it. Otherwise create one.
     if (existingInteraction) {
-      existingInteraction.entries.push(entry);
-      existingInteraction.latency = Math.max(
-        existingInteraction.latency,
-        entry.duration,
-      );
+      if (entry.duration > existingInteraction.latency) {
+        existingInteraction.latency = entry.duration;
```

Comment:
I know this isn't new code, but, is the term `latency` exposed in any way?  Feels inconsistent.

Reply:
This usage of the term "latency" matches the definitions originally proposed here: https://web.dev/blog/responsiveness#measure_interaction_latency

Also see the definitions from this Google-internal [doc](https://docs.google.com/document/d/1mjbw-5fDNOBaDOjyz6kq54ayj6QHzNbGXtID4UpgVWw/edit?resourcekey=0-G2oQ0rOGyK9OJASVxz49TQ&tab=t.0#bookmark=id.lbfs6zy3rnb6).

Reply:
> is the term `latency` exposed in any way? Feels inconsistent.

To directly answer you question though, no this is not exposed in any way. It's an implementation detail.